include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -21,7 +21,8 @@ class VerifOp<string mnemonic, list<Trait> traits = []> :
 
 class AssertLikeOp<string mnemonic, list<Trait> traits = []> :
   VerifOp<mnemonic, traits> {
-  let arguments = (ins LTLAnyPropertyType:$property);
+  let arguments = (ins LTLAnyPropertyType:$property,
+                       OptionalAttr<StrAttr>:$label);
   let assemblyFormat = [{
     $property attr-dict `:` type($property)
```

Comment:
Could you put $label into assembly format?

Reply:
Good point! Done :+1: